vueui/src/components/view/medical/bodyscanner.vue

```diff
@@ -0,0 +1,285 @@
+            <vui-group-item label="Norepinephrine:" v-if="round(norepiAmt)"> {{ round(norepiAmt) }} unit(s)</vui-group-item>
+            <vui-group-item label="Soporific:" v-if="round(soporAmt)"> {{ round(soporAmt) }} unit(s)</vui-group-item>
+            <vui-group-item label="Bicaridine:" v-if="round(bicardAmt)"> {{ round(bicardAmt) }} unit(s)</vui-group-item>
+            <vui-group-item label="Dermaline:" v-if="round(dermAmt)"> {{ round(dermAmt) }} unit(s)</vui-group-item>
+            <vui-group-item label="Dexalin:" v-if="round(dexAmt)"> {{ round(dexAmt) }} unit(s)</vui-group-item>
+            <vui-group-item label="Other:" v-if="round(otherAmt)"> {{ round(otherAmt) }} unit(s)</vui-group-item>
+          </vui-group>
+        </div>
+
+        <!-- Right Side -->
+        <div class="column">
+
+          <!-- Missing Organs -->
+          <template v-if="hasmissing">
+            <div v-for="organ in missingparts" :key="organ.name">
+              <vui-group-item> {{ organ }} </vui-group-item>
+            </div>
+          </template>
+
+          <!-- Internal Organs -->
+          <h3>Internal Organ Status</h3>
+          <hr>
+          <table>
+            <tr>
+              <th>Organ</th>
+              <th>Trauma</th>
+              <th>Wounds</th>
+            </tr>
+            <tr v-for="organ in organs" :key="organ.name">
+              <template v-if="organ.damage != 'none' || organ.hasWounds">
+                <td> {{ capitalizeFirstLetter(organ.name) }} </td>
+                <td><span :style="{color:damageLabel(organ.damage)}"> {{ organ.damage }}</span></td>
+                <td><div v-for="wound in organ.wounds" :key="wound.name" class="bad"> {{ wound }} </div></td>
+              </template>
+            </tr>
+          </table>
+
+          <!-- External Organs -->
+          <hr>
+          <h3>External Bodypart Status</h3>
+          <hr>
+          <table>
+            <tr>
+              <th>Organ</th>
+              <th>Physical / Burn Trauma</th>
+              <th>Wounds</th>
+            </tr>
+            <tr v-for="organ in bodyparts" :key="organ.name">
+              <template v-if="organ.bruteDmg != 'none' || organ.burnDmg != 'none' || organ.hasWounds">
+                <td> {{ capitalizeFirstLetter(organ.name) }} </td>
+                <td><span :style="{color:damageLabel(organ.bruteDmg)}"> {{ organ.bruteDmg }} </span> / <span :style="{color:damageLabel(organ.burnDmg)}"> {{ organ.burnDmg }} </span></td>
+                <td><div v-for="wound in organ.wounds" :key="wound.name"> {{ wound }} </div></td>
+              </template>
+            </tr>
+          </table>
+        </div>
+
+        <h3>Actions</h3>
+        <vui-button class="center" :params="{print: 1}">Print Report</vui-button>
+
+      </div>
+    </div>
+  </div>
+</template>
+
+
+<script>
+  export default {
+    data() {
+      return this.$root.$data.state;
+    },
+    methods: {
+    round(value) {
+      return Math.round(value)
```

Comment:
What is this indentation. Also why you couldn't use `Math.round()` directly in template?